game/consumers.py

```
@@ -0,0 +1,44 @@
+            )
+
+            await self.channel_layer.group_send(
+                "online_users",
+                {
+                    "type": "status_update",
+                    "username": user.username,
+                    "status": "online"
+                }
+            )
+
+    async def disconnect(self, close_code):
```

Ruff (0.15.12)

[warning] Unused method argument: `close_code`

(ARG002)